Pipfile

```diff
@@ -1,62 +1,78 @@
+# RERO ILS
+# Copyright (C) 2020 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
 [[source]]
 url = "https://pypi.org/simple"
 verify_ssl = true
 name = "pypi"
 
 [packages]
-# TODO: try to remove it when werkzeug will be fixed.
-werkzeug = "==0.16.0"
-# TODO: check why we need this.
-flask-login = "<0.5"
+## Python packages dependencies (order matters)
+#----------------------------------------------
```

Comment:
Is that possible to explain a little bit why these paragraphs are in a certain order?
Especially why we put dependancies from invenio metadata bundle before invenio ;  why we put invenio-records after (for example), etc.